src/vss_tools/model.py

```diff
@@ -178,6 +178,10 @@ class VSSDataDatatype(VSSData):
     arraysize: int | None = None
     min: int | float | None = None
     max: int | float | None = None
+    # Field, used to allow definition of Regular Expression constraints
+    # for string based property nodes.
+    # Example: VSS - VehicleIdentification.VIN property
+    pattern: str | None = None
```

Comment:
We would also need a model validator function for `pattern` to evaluate it. It should fail when being defined and:
- `datatype` is  not a string
- `default`  being set and is not matching

Reply:
Thank you Sebastian, I will update the model.

Reply:
Done